code/stable/pdcontroller/src/cpp/Makefile

```diff
@@ -12,18 +12,18 @@ endif
 
 # Generated by Drasil v0.1-alpha
 
-# Project Name: PD_Controller
+# Project Name: PDController
```

Comment:
Again, similar to the above comment, using the same uniform naming scheme that we use in URIs, etc.